app/algorithms/normal.ts

```diff
@@ -65,26 +66,25 @@ const calculateWinnerUsingNormal = (
   const thr = winningThreshold(votes, useStrict);
 
   // Winner key
-  const maxKey: string = Object.keys(count).reduce((a, b) =>
+  const maxKey = Object.keys(count).reduce((a, b) =>
     count[a] > count[b] ? a : b
   );
 
   // Check if we can call the vote Resolved based on
-  const status = count[maxKey] >= thr ? Status.resolved : Status.unresolved;
-
-  // Create winner alternative from maxKey
-  const winner =
-    count[maxKey] >= thr
-      ? {
-          ...alternatives.find((a) => a.description === maxKey),
-          count: count[maxKey],
-        }
-      : undefined;
+  let status: Status = Status.unresolved;
+  let winner: Alternative = undefined;
+  if (count[maxKey] >= thr) {
+    winner = {
+      ...alternatives.find((a) => a.description === maxKey),
+      count: count[maxKey],
+    } as Alternative;
```

Comment:
Hmm, I don't like this type casting

Reply:
Oh damn :o
Removed casting and added it as an optional field in the Alternative-type 👍